src/Microsoft.ML.OnnxTransformer/OnnxUtils.cs

```diff
@@ -439,7 +441,15 @@ public static NamedOnnxValue CreateScalarNamedOnnxValue<T>(string name, T data)
         {
             if (!_onnxTypeMap.Contains(typeof(T)))
                 throw new NotImplementedException($"Not implemented type {typeof(T)}");
-            return NamedOnnxValue.CreateFromTensor<T>(name, new DenseTensor<T>(new T[] { data }, new int[] { 1 }));
+
+            if (data.GetType() == typeof(ReadOnlyMemory<Char>))
+            {
+                var charMemory = (ReadOnlyMemory<Char>)Convert.ChangeType(data, typeof(ReadOnlyMemory<Char>));
```

Comment:
You don't need `Convert.ChangeType`. You can just cast `data` to `ReadOnlyMemory<char>`.

Reply:
That doesn't seem to work for me. I get this error:

Cannot convert type 'T' to 'System.ReadOnlyMemory<char>'

---
In reply to: [340779381](https://github.com/dotnet/machinelearning/pull/4416#discussion_r340779381) [](ancestors = 340779381)